README.md

```diff
@@ -39,6 +39,32 @@ Before you get started:
   * `~/.git-credentials` must exist with git credentials. If you are using two-factor auth you will need a create a [personal access token](https://help.github.com/en/articles/creating-a-personal-access-token-for-the-command-line)
   * `lint` in $PATH which is provided by having `$GOPATH\bin` in `$PATH` as in `export PATH=$GOPATH/bin:$PATH`.
 
+## Running the Server
```

Comment:
Why would we call it `Server` - shouldn't it be more `## Running the Operator` ?